app/helpers/template_helpers.py

```diff
@@ -79,7 +79,7 @@ def render_template(template, **kwargs):
         languages=get_languages_context(),
         schema_theme=theme,
         language_code=get_locale().language,
-        survey_title=cookie_session.get("survey_title"),
+        survey_title=get_survey_title(cookie_session.get("survey_title")),
```

Comment:
There is an issue with getting the survey title from the cookie - it's not updated when the user changes language. Unfortunately the fix isn't as simple as updating the cookie when they change language, as some routes don't have access to the schema e.g. errors. 

I think given that the survey title was already hard-coded to "Census 2021" in some places, we should just use that here (and raise a card to investigate survey titles for all schemas).

```suggestion
        survey_title=lazy_gettext("Census 2021"),
```

Reply:
I had to change all tests to achieve this, I hope it's ok with you? Tests schemas have a "Census 2021" suffixed page titles now.